package.json

```diff
@@ -53,4 +53,4 @@
       "git add"
     ]
   }
-}
+}
```

Comment:
Why no new line?

Reply:
look like lint is not apply to `.json`

Reply:
Should need as only file that is json is `package.json`. Self review should be enough.